public/app/features/explore/ExploreTimeControls.tsx

```diff
@@ -92,6 +92,7 @@ export class ExploreTimeControls extends Component<Props> {
         {...timePickerCommonProps}
         timeSyncButton={timeSyncButton}
         isSynced={syncedTimes}
+        widthOverride={splitted ? window.innerWidth / 2 : undefined}
```

Comment:
@glintik is this what you had in mind? this was my mistake - thanks for catching it!